tools/cockpit.spec

```diff
@@ -101,15 +101,16 @@ Requires: cockpit-ws
 Requires: cockpit-system
 
 # Optional components
+Recommends: (cockpit-storaged if udisks2)
+Recommends: cockpit-packagekit
+Suggests: cockpit-pcp
+
 %if 0%{?rhel} == 0
 Recommends: cockpit-dashboard
 %ifarch x86_64 %{arm} aarch64 ppc64le i686 s390x
 Recommends: (cockpit-docker if /usr/bin/docker)
 %endif
```

Comment:
heh, I was looking at it for a bit until I realized that this `endif` is connected with `ifarch` :)